pkg/smartcontract/nef/nef.go

```diff
@@ -48,10 +46,9 @@ type File struct {
 
 // Header represents File header.
 type Header struct {
-	Magic      uint32
-	Compiler   string
-	Version    Version
-	ScriptHash util.Uint160
+	Magic    uint32
+	Compiler string
+	Version  Version
```

Comment:
Version is finally a string!

Reply:
Very recent, but welcome change for sure, I'll update this.